### ISSUE TITLE
missing code

### ISSUE THREAD

**Issue by KK4HFJ** [Wireframe-Magazine/Wireframe-46#1]
two lines of missing code.
very first line:
import pgzrun
very last line:
pgzrun.go()

now "python pipemania.py" works.

**Comment by Scubadooper** [comment]
I found that code wasn't needed

**Comment by dww100** [comment]
The code is not needed if you run using:

```
pgzrun PipeMania.py
```

If however you want to run in the way @KK4HFJ shows then it is. I think this is more a documentation than code issue.

**Comment by SheetLightning** [comment]
Yes, the assumption is being made that the user is familiar with pgzero knows how to run the code. I was just getting 'Hello from the pygame community.' until I read this issue and added those lines.

Thre is also an error in the printed version in line 20. The part '_path.png' should be '_way.png'.
